src/Bling.js

```diff
@@ -642,7 +642,8 @@ class Bling extends Component {
             if (outOfPage) {
                 this._adSlot = Bling._adManager.googletag.defineOutOfPageSlot(
                     adUnitPath,
-                    divId
+                    Bling._adManager.googletag.enums.OutOfPageFormat
+                        .INTERSTITIAL
```

Comment:
Add comment here that we may need additional logic when we have other OOP types?